psycopg/psycopg/connection_async.py

```diff
@@ -90,6 +91,20 @@ async def connect(
         row_factory: Optional[AsyncRowFactory[Row]] = None,
         **kwargs: Any,
     ) -> "AsyncConnection[Any]":
+
+        if sys.platform == "win32":
+            loop = asyncio.get_running_loop()
+            if isinstance(loop, asyncio.ProactorEventLoop):
+                # TODO: is there a general psycopg error not related
+                # to db operations? InterfaceError
+                raise NotImplementedError(
+                    "psycopg does not currently support running in async mode "
+                    "on windows on the ``ProactorEventLoop`` event loop. "
```

Comment:
`'ProactorEventLoop' event loop`  is redundant in a message already quite long :)